README.md

```diff
@@ -0,0 +1,357 @@
+# Rabbit MQ Topic ACL Authorization plugin
+
+## Index
+
+* [Overview](#overview)
+* [Usage](#usage)
+  * [Deployment](#deployment)
+  * [Administration](#administration)
+  * [Examples of use](#examples)
+* [Development](#development)
+  * [Internal Architecture](#architecture)
+  * [Build](#build)
+  * [Test](#test)
+
+## <a name="overview"/> Overview
+### Description
+This project aims to provide an authorization plugin for Rabbit MQ that will manage the access to RabbitMQ resources
+based on the Routing Keys used to publish and bind queues. The ultimate goal of this plugin is to be used along the
+RabbitMQ MQTT plugin in order to use RabbitMQ as an MQTT broker with ACL-based authorization.
+
+Permissions for each routing key will be defined in Access Control Lists (ACLs) with a format similar to the one used
+for Mosquitto ACLs.
+
+## <a name="usage"/> Usage
+### <a name="deployment"/> Deployment
+
+In order to deploy the plugin:
+- Build the packages for distribution with `make dist`.
+- Move the generated `plugins/rabbitmq_topic_acl-<version>.ez`
+- Enable the plugins with the `rabbitmq-plugins` administration command.
```

Comment:
It is a shell command (vs. a comment inside a "rabbit shell" if that things make sense :)?